src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/AssessmentOrgsRepository.cs

```diff
@@ -90,36 +93,32 @@ public IEnumerable<Organisation> GetOrganisationsByStandardId(string standardId)
                 throw new ApplicationException($"Failed query organisations by standard id");
             }
 
-            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date);
+            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date && (x.EffectiveTo >= DateTime.UtcNow.Date || x.EffectiveTo == null)).ToList();
```

Comment:
It's ok if there's empty EffectiveFrom date, since there's some in there already. we can talk to richard about removing those before we add a test to prevent it